app/src/main/java/com/jerboa/db/repository/AppSettingsRepository.kt

```diff
@@ -40,16 +38,12 @@ class AppSettingsRepository(
     }
 
     @WorkerThread
-    suspend fun updateChangelog() {
+    suspend fun updateChangelog(ctx: Context) {
         withContext(Dispatchers.IO) {
             try {
-                Log.d("jerboa", "Fetching RELEASES.md ...")
-                // Fetch the markdown text
-                val releasesUrl =
-                    "https://raw.githubusercontent.com/dessalines/jerboa/main/RELEASES.md".toHttpUrl()
-                val req = Request.Builder().url(releasesUrl).build()
-                val res = API.httpClient.newCall(req).execute()
-                _changelog.value = res.body.string()
+                Log.d("jerboa", "Getting RELEASES.md from assets...")
+                val releasesStr = ctx.assets.open("RELEASES.md").bufferedReader().use { it.readText() }
```

Comment:
I rather have this logic in the viewmodel. Just pass a stream or string.

I see that we did this before but we shouldn't have

Reply:
I'll try to refactor that now.